presto-delta/src/main/java/com/facebook/presto/delta/DeltaModule.java

```diff
@@ -113,7 +116,8 @@ protected void setup(Binder binder)
 
         configBinder(binder).bindConfig(HiveGcsConfig.class);
         binder.bind(GcsConfigurationInitializer.class).to(HiveGcsConfigurationInitializer.class).in(Scopes.SINGLETON);
-
+        configBinder(binder).bindConfig(HiveAzureConfig.class);
+        binder.bind(AzureConfigurationInitializer.class).to(HiveAzureConfigurationInitializer.class).in(Scopes.SINGLETON);
```

Comment:
As we have already created `HiveAzureModule`, please re-use that. You can bind it in `DeltaConnectorFactory`